### PR TITLE
Add Custom Attributes section in core version with pro promotion

### PR DESCRIPTION
**Pull request by udidol**
## Quality assurance

- [X] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: includes/managers/controls.php [hunk 1]
```diff
@@ -913,4 +913,50 @@ public function add_custom_css_controls( Controls_Stack $controls_stack ) {
 
 		$controls_stack->end_controls_section();
 	}
+
+	/**
+	 * Add custom Attributes controls.
```
Comment (matipojo) [inline includes/managers/controls.php]:
```suggestion
	 * Add custom attributes controls.
```

File: includes/managers/controls.php [hunk 2]
```diff
@@ -913,4 +913,50 @@ public function add_custom_css_controls( Controls_Stack $controls_stack ) {
...
+	 *
+	 * This method adds a new control for the "Custom Attributes" feature. The free
+	 * version of elementor uses this method to display an upgrade message to
+	 * Elementor Pro.
+	 *
+	 * @since 1.0.0
```
Comment (matipojo) [inline includes/managers/controls.php]: Remove
```suggestion
	 * @since 2.8.3
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/managers/controls.php [hunk 3]
```diff
@@ -913,4 +913,50 @@ public function add_custom_css_controls( Controls_Stack $controls_stack ) {
...
+				'label' => __( 'Attributes', 'elementor' ),
+				'tab' => self::TAB_ADVANCED,
+			]
+		);
+
+		$controls_stack->add_control(
+			'custom_attributes_pro',
+			[
+				'type' => self::RAW_HTML,
+				'raw' => '<div class="elementor-nerd-box">' .
```
Comment (kobizz) [inline includes/managers/controls.php]: Make a general method that retrieves the template and use it for both custom-css and custom attributes

**Review by kobizz — CHANGES_REQUESTED** [review]

File: includes/managers/controls.php [hunk 4]
```diff
@@ -893,21 +893,73 @@ public function add_custom_css_controls( Controls_Stack $controls_stack ) {
...
+					],
+					'link' => 'https://elementor.com/pro/?utm_source=panel-custom-css&utm_campaign=gopro&utm_medium=wp-dash',
+				] ),
+			]
+		);
+
+		$controls_stack->end_controls_section();
+	}
+
+	public function get_teaser_template( $texts ) {
```
Comment (kobizz) [inline includes/managers/controls.php]: Use buffering

**Review by kobizz — CHANGES_REQUESTED** [review]

**Review by kobizz — APPROVED** [review]

**Pull request merged**